src/rf/apps/uploads/views.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.contrib.auth.models import User
+
+from rest_framework import serializers, viewsets
+from rest_framework.response import Response
+from rest_framework import decorators
+from rest_framework.permissions import AllowAny
+from rest_framework.renderers import StaticHTMLRenderer
+
+from django.conf import settings
+from django.shortcuts import render_to_response
+
+import base64
+import hmac, sha
+
+
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+@decorators.renderer_classes((StaticHTMLRenderer, ))
```

Comment:
I'm also using class-based views, so you might want to change it to be consistent.